event-handlers/src/main/java/no/sikt/nva/nvi/events/UpsertNviCandidateHandler.java

```diff
@@ -60,8 +67,44 @@ private static void validateRequiredCandidateDetails(CandidateDetails candidateD
         Objects.requireNonNull(candidateDetails.verifiedCreators());
     }
 
+    private static DbCandidate toPendingCandidate(CandidateEvaluatedMessage message) {
```

Comment:
method name`toCandidate`? Not generating pending approval statuses here any more

Reply:
toPendingDbCandidate / toDbCandidate?